tests/unit/LocalDB.js

```diff
@@ -253,9 +253,10 @@ define([
 					]);
 				});
 			},
-			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))":
-				testQuery(new Filter().in('mappedTo', letterStore.filter({vowel: true}).select('id')),
-					{sort:[{property: 'name'}], range: {start: 1, end: 3}}, [5, 1]),
+			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))": (function() {
```

Comment:
I don't think this change needs to be made whatsoever.  `(function () { return testQuery(...); })()` should be identical to just `testQuery(...)` which is what it was before.